composer.json

```
@@ -29,7 +29,7 @@
         "drupal/drupal_cms_page": "^1.1",
         "drupal/drupal_cms_privacy_basic": "^1.1",
         "drupal/eca": "^2.1.3",
-        "drupal/experience_builder": "^0.2.0-alpha6@alpha",
+        "drupal/experience_builder": "0.3.0-alpha1",
```

```suggestion
        "drupal/experience_builder": "0.3.0-alpha1@alpha",
```
Won't work without the stability flag, since `minimum-stability` is `stable`.